R/init-package.R

```diff
@@ -17,6 +17,7 @@ initPackage <- function() {
   apiConfigNet <- clrNew("OSPSuite.R.ApiConfig")
   apiConfig <- ApiConfig$new(apiConfigNet)
   apiConfig$dimensionFilePath <- filePathFor("OSPSuite.Dimensions.xml")
+  apiConfig$pkParametersFilePath <- filePathFor("OSPSuite.PKParameters.xml")

```

Comment:
This file is the generic files that describes the pkparameters. Required for calculation (similar to PKSim and Mobi implementation)